crates/mdbook-goals/src/mdbook_preprocessor.rs

```diff
@@ -365,4 +378,63 @@ impl<'c> GoalPreprocessorWithContext<'c> {
         }
         Ok(())
     }
+
+    /// Replace TEAMS_WITH_ASKS placeholder with a list of teams.
```

Comment:
Since this handles more than `TEAMS_WITH_ASKS` maybe don't mention it _here_, or use that as a comment for the associated `replace_metadata_placeholder` call?

e.g.
```suggestion
    /// Replace placeholders like TASK_OWNERS and TEAMS_WITH_ASKS.
```